src/components/MultiSelect/MultiSelectMenuButton.tsx

```diff
@@ -24,11 +19,8 @@ export interface MultiSelectMenuButtonProps {
   /** The action to perform for clear/reset button of MultiSelect. */
   onClear?: () => void;
   onKeyDown?: () => void;
+  buttonRef: any,
```

Comment:
This component is a button as well so this could use a better prop name because it's confusing.